AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1468,18 +1455,9 @@ - (void)__layout
   [self layoutDidFinish];
 }
 
+/// Needs to be called with lock held
```

Comment:
Let's start with `_locked_measure...` to be aligned with PINRemoteImage and PINCache